scripts/check_dep_list.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 Preferred Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Validates that DEPENDENCIES.md is synced with Gopkg.toml.
+"""
+
+from pathlib import Path
+
+PROJECT_ROOT = Path(__file__).resolve().parents[1]
+
+GOPKG_PATH = "Gopkg.toml"
+DEPLIST_PATH = "DEPENDENCIES.md"
+
+WHITELIST_GOPKG = [  # may be missing in Gopkg.toml
+    "k8s.io/api",
+    "k8s.io/apimachinery",
+]
+
+WHITELIST_DEPLIST = [  # may be missing in DEPENDENCIES.md
+]
+
+
+def main(verbose=False):
+    ok = True
+
+    gopkg = deps_gopkg()
+    if verbose:
+        print("Dependencies in " + GOPKG_PATH + ":")
+        for g in gopkg:
+            print("-", g)
+        print()
+
+    deplist = deps_list()
+    if verbose:
+        print("Dependencies in " + DEPLIST_PATH + ":")
+        for d in deplist:
+            print("-", d)
```

Comment:
can we print them with something like `print("- %40s %20s" % d)` for better readability?